### PR TITLE
[READY] - [inf-issue522] - add README for term-apply

### PR DESCRIPTION
**Pull request by kylerisse**
Adds a README for term-apply

Previous:
- no docs for term-apply

New:
- docs for term-apply focused on local development and usage

Test:
- [x] review the [rendered markdown](https://github.com/Nebulaworks/orion/blob/kr/inf-issue-522_term-apply-readme/apps/term-apply/README.md)

### CODE REVIEW

File: apps/term-apply/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## Development
+
+These instructions recommend using `nix-shell`. If you choose not to, please make sure you have a functional `go 1.17` installation and the `make` command installed.
+
+1. You will need access to an S3 bucket and prefix(es) with the following object permissions.
+
+```
+s3:GetObject
+s3:PutObject
+s3:PutObjectTagging
```
Comment (sarcasticadmin) [inline apps/term-apply/README.md:18]: Thanks for calling these out specifically

File: apps/term-apply/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+4. If necessary, set other [environment variables](#environment-variables) for your specific environment.
+> For most usage, the defaults are fine for local development
+
+5. Create an `uploads` directory if one does not yet exist
+```
+mkdir uploads
+```
+
+6. Code
```
Comment (sarcasticadmin) [inline apps/term-apply/README.md:39]: most important :wink:

File: apps/term-apply/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+7. Compile the code
+```
+make test && make build
```
Comment (sarcasticadmin) [inline apps/term-apply/README.md:43]: This made me realize that `GNUmake` wasnt in the nix-shell: https://github.com/Nebulaworks/orion/blob/main/apps/term-apply/shell.nix#L8-L11

I think itd be a good idea to add it for completeness: https://github.com/Nebulaworks/orion/blob/main/apps/term-apply/shell.nix#L8-L11

File: apps/term-apply/README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+8. Test
+```
+./term-apply
+```
+
+## Environment Variables
```
Comment (sarcasticadmin) [inline apps/term-apply/README.md:51]: :heart: the call out here for env vars

File: apps/term-apply/README.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+# term-apply
+
+term-apply is a custom SSH server used by candidates to submit their resumes
+
+## Usage
+
+term-apply is maintained and distriubuted as a [nix package](https://github.com/Nebulaworks/nix-garage/blob/master/pkgs/term-apply/default.nix) and [container](https://hub.docker.com/r/nebulaworks/term-apply).
```
Comment (sarcasticadmin) [inline apps/term-apply/README.md:7]: thanks for calling out the relationship with nix-garage

**Review by sarcasticadmin — APPROVED** [review]
@kylerisse looks great!

I had one comment about the nix-shell, let me know what you think and then Ill merge

File: apps/term-apply/README.md (hunk 3 above)
Comment (kylerisse) [reply apps/term-apply/README.md:43]: nice catch, yes it should be. i mistakenly assumed it already was because of *some other project*

**Comment by kylerisse** [comment]
@sarcasticadmin ready for merge, thanks for the review!

**Comment by sarcasticadmin** [comment]
:shipit:

**Pull request merged**